concourse/client/model.py

```diff
@@ -326,6 +375,24 @@ def name(self):
         return self.raw['name']
 
 
+class PipelineResource:
+    '''
+    Wrapper around the dictionary representing a pipeline resource
+
+    Not intended to be instantiated by users of this module
+    '''
+    @ensure_annotations
+    def __init__(self, raw:dict, concourse_api):
+        self.concourse_api = concourse_api
+        self.raw = raw
+        self.name = raw['name']
+
+    def has_pinned_version(self):
```

Comment:
nit: rather name this `is_pinned`? as this is the semantics that this means

also, it might be useful to expose the `pinned_version`, actually (e.g. to check if the pinned version is maybe the one we would like to pin in the first place)